src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -10727,8 +10727,8 @@ void M(string s)
                 }
             }
             """,
-            // PROTOTYPE: There should be no overloads here.  Need GetMemberGroup fixed for skeleton vs impl extensions.
-            MainDescription($"void Extensions.extension(string).Goo() (+ 1 {FeaturesResources.overload})"));
+
+            MainDescription($"void Extensions.extension(string).Goo()"));
```

Comment:
How was this fixed in this PR?

Reply:
> How was this fixed in this PR?

It was "fixed" by not calling ```GetMembersUnordered``` in ```SourceMemberContainerSymbol.ContainsExtensionMethods``` and running into a pre-existing bug that "masked" the wrong behavior.